components/apitemplate_io/actions/account-information/account-information.ts

```diff
@@ -0,0 +1,24 @@
+import app from "../../app/apitemplate_io.app";
+import { defineAction } from "@pipedream/types";
+
+export default defineAction({
+  name: "Account Information",
+  description: "Retrieves information about your account. [See the docs](https://apitemplate.io/apiv2/) for more information",
```

Comment:
I could not find `/account-information` endpoint in [the doc](https://apitemplate.io/apiv2/). If possible, can you please use an anchor link or another link here?

Reply:
it's not documented, it was asked by on the task and I found out the API URL, but there's no documentation about it.